apps/taquito-test-dapp/src/tests.ts

```diff
@@ -551,6 +551,8 @@ export const init = (
     id: "send-tez",
     name: "Send tez",
     description: "This test sends 0.1 tez to Alice's address",
+    documentation: 'https://tezostaquito.io/docs/quick_start#transfer',
```

Comment:
Some documentation pages have more information than the "quick start". Consider using the following link instead: https://tezostaquito.io/docs/wallet_API#making-transfers